lib/slots.ts

```diff
@@ -32,12 +84,19 @@ const getSlots = ({
     for (
       let minutes = dayStartTime;
       minutes <= dayEndTime - eventLength;
-      minutes += parseInt(eventLength, 10)
     ) {
       const slot = lowerBound.add(minutes, "minutes");
+
       if (slot > now) {
-        slots.push(slot);
+        const slotAvailablilty = isSlotAvailable({ slotStartTime: slot, slotEndTime: slot.add(eventLength, 'minutes'), busyTimes });
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689309)
```suggestion
        const slotAvailablilty = isSlotAvailable({ slotStartTime: slot, slotEndTime: slot.add(eventLength, "minutes"), busyTimes });
```